recipes-kernel/linux/linux-compulab-common.inc

```diff
@@ -0,0 +1,19 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/linux-compulab:"
+
+SRC_URI:append = " \
+    file://crypto-engine-fix-crypto_queue-backlog-handling.patch \
+"
+
+BALENA_CONFIGS:append = " caam"
+BALENA_CONFIGS[caam] = " \
+    CONFIG_DAX=y \
+    CONFIG_BLK_DEV_DM=y \
+    CONFIG_BLK_DEV_MD=y \
+    CONFIG_MD=y \
```

Comment:
Nitpick: Do you really need `MD` and `BLK_DEV_MD`? That should only be necessary for RAID and LVM.

Reply:
It is according to the documentation required for dm-crypt:
![image](https://github.com/user-attachments/assets/40c2dea2-97a9-4994-83c7-be23c1e6ce24)